indra/sources/bel/processor.py

```diff
@@ -786,9 +784,12 @@ def _get_translocation_target(node_modifier_data):
     to_loc_ns = to_loc_info.get(pc.NAMESPACE)
     to_loc_name = to_loc_info.get(pc.NAME)
     # Only use GO Cellular Component location names
-    if to_loc_ns != 'GOCC' or not to_loc_name:
+    if to_loc_ns not in ('GOCC', 'GOCCID') or not to_loc_name:
```

Comment:
Should this also accept `'GO'`?

Reply:
I guess that wouldn't hurt, I haven't come across that yet but I'll add it